File: Hackerrank — 30 days of code/day_20.py
#!/bin/python3

n = int(input().strip())
a = [int(a_temp) for a_temp in input().strip().split(' ')]

numSwaps = 0
for i in range(n):
    for j in range(i + 1, n):
        if a[i] > a[j]:
            a[i], a[j] = a[j], a[i]
            numSwaps += 1

print('Array is sorted in {} swaps.'.format(numSwaps))
print('First Element:', a[0])
print('Last Element:', a[n - 1])

File: Hackerrank — 30 days of code/day_20.java
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int a[] = new int[n];
        for(int a_i=0; a_i < n; a_i++){
            a[a_i] = in.nextInt();
        }
        int count = 0;
        for(int i = 0; i < n - 1; i++) {
            for(int j = i + 1; j < n; j++) {
                if(a[i] > a[j]) {
                    int temp = a[i];
                    a[i] = a[j];
                    a[j] = temp;

                    count++;
                }
            }
        }

        System.out.printf("Array is sorted in %d swaps.\n", count);
        System.out.println("First Element: " + a[0]);
        System.out.println("Last Element: " + a[n - 1]);
    }
}
